pkg/apis/provisioning/v1alpha3/provisioner.go

```diff
@@ -113,11 +113,6 @@ var (
 )
 
 var (
-	// Well known labels
-	ArchitectureLabelKey    = "kubernetes.io/arch"
-	OperatingSystemLabelKey = "kubernetes.io/os"
-	ZoneLabelKey            = "topology.kubernetes.io/zone"
-	InstanceTypeLabelKey    = "node.kubernetes.io/instance-type"
```

Comment:
can you point me to the replacement for this? does it exist?

Reply:
https://github.com/kubernetes/api/blob/master/core/v1/well_known_labels.go

Reply:
ah, that's what I was missing, it's not in our github repo, it's imported. correct?